lib/browser/guest-view-manager.js

```diff
@@ -160,24 +151,24 @@ const attachGuest = function (embedder, elementInstanceId, guestInstanceId, para
     destroyGuest(embedder, oldGuestInstanceId)
   }
 
-  guestInstance = guestInstances[guestInstanceId]
+  const guestInstance = guestInstances[guestInstanceId]
   // If this isn't a valid guest instance then do nothing.
   if (!guestInstance) {
     return
```

Comment:
Yeah, I wasn't planning on touching these, I'd be up for doing them in a secondary pass though